v2/pkg/options/options.go

```diff
@@ -165,6 +169,31 @@ func MergeDefaults(appoptions *App) {
 	processDragOptions(appoptions)
 }
 
+type SingleInstanceLock struct {
+	// uniqueId that will be used for setting up messaging between instances
+	UniqueId               string
+	Enabled                bool
```

Comment:
Is there a scenario where this is false? As we are using a pointer to the struct, we could assume if it's available then the intent is to enable it?

Reply:
Just had in mind to make it more explicit, but we can just check that pointer to the struct exists. Happy to change it